doc/config.md

```diff
@@ -51,6 +51,11 @@ Clj-kondo can be configured in several ways:
 - project config: a `config.edn` file in the `.clj-kondo` directory (see
   [project setup](../README.md#project-setup)).
 - `:config-paths` in project `config.edn`: a list of directories that provide additional config.
+- `:extra-config-paths` in project `config.edn`: a list of directories that provide additional config
```

Comment:
Should we still have the `:extra-config-paths` if we have the environment variable?

Is it easy to set the environment variable for you editor on a project by project basis, have you tested this?